app/models/data_update_script.rb

```diff
@@ -12,7 +12,7 @@ class << self
     def scripts_to_run
       insert_new_scripts
 
-      enqueued.where(id: ids).order(file_name: :asc)
```

Comment:
Ummm, I have no idea why we did it this way but great catch!!!

Reply:
Git blame told me that `ids` was originally meant to be load_script_ids below :)

```ruby
    def load_script_ids
      filenames.
        map { |file_name| find_or_create_by(file_name: file_name) }.
        select(&:enqueued?).
        map(&:id)
    end
```